src/commands/add.ts

```diff
@@ -421,8 +412,23 @@ const fieldSubCommand = async (ctx: Context, { args, filesync }: { args: AddArgs
     }
   }
 
-  println({
-    ensureEmptyLineAbove: true,
-    content: `Field ${chalk.cyanBright(modelFieldsList[0]?.name)} added successfully.`,
+  println({ ensureEmptyLineAbove: true, content: `Environment ${chalk.cyanBright(newEnvName)} added successfully.` });
+
+  // Try to switch to newly made env
+  const pullFromNewEnvSyncJson = await SyncJson.load(ctx, {
+    command: "pull",
+    args: { _: [], "--app": undefined, "--allow-unknown-directory": undefined, "--allow-different-app": undefined, "--env": newEnvName },
+    directory: await loadSyncJsonDirectory(process.cwd()),
   });
+  if (pullFromNewEnvSyncJson) {
+    await runPull(ctx, pullFromNewEnvSyncJson, true);
+  }
+};
```

Comment:
`pull.ts` delegates most of its logic to `filesync.pull` so I would just copy/paste what `pull.ts` does rather than import from it, e.g.

```ts
// Try to switch to newly made env
const newEnvSyncJson = await SyncJson.load(ctx, {
  command: "pull",
  args: { ...args, "--env": newEnvName },
  directory: filesync.syncJson.directory,
});

if (newEnvSyncJson) {
  const newEnvFileSync = new FileSync(newEnvSyncJson);
  await newEnvFileSync.pull(ctx, { force: true });
}
```